src/app/shared/issue-tables/IssuesDataTable.ts

```diff
@@ -162,8 +162,18 @@ export class IssuesDataTable extends DataSource<Issue> {
   }
 
   private compareValue(valueA, valueB): number {
-    const a = valueA || '';
-    const b = valueB || '';
+    let a: string | number;
+    let b: string | number;
+    if (typeof valueA === 'string') {
+      a = valueA.toUpperCase();
+    } else {
+      a = valueA || '';
+    }
+    if (typeof valueB === 'string') {
+      b = valueB.toUpperCase();
+    } else {
+      b = valueB || '';
+    }
```

Comment:
If I am not wrong, we can type cast numbers to string. (e.g. `String(24)`) The comparison should work the same way as comparing actual numbers. Doing so can simplify the code quite a bit.